src/components/Todo.js

```diff
@@ -0,0 +1,19 @@
+
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faPenToSquare } from '@fortawesome/free-regular-svg-icons'
+import { faTrash } from '@fortawesome/free-solid-svg-icons'
+
+export const Todo = ({ task, toggleComplete, deleteTodo, editTodo }) => {
+    return (
+        <div className='Todo'>
+            <p onClick={()=> toggleComplete(task.id)} className={`${task.completed ? 'completed' : ''}`}>{task.task}</p>
```

Comment:
you dont need to do this
className={`${task.completed ? 'completed' : ''}`}
you can just do this
className={task.completed ? 'completed' : ''}